### DISCUSSION TITLE
Debian 12 stable cannot restart system (Login Screen for Wayland)

### DISCUSSION THREAD

**Discussion by apotenza92** [rustdesk/rustdesk#7155] [General]
### Bug Description

Hi, I'm on Debian 12 and cannot get rustdesk to reconnect after a system reboot. I have to login manually.
I have tried changing Permissions to be full access and include 'Enable remote configuration modification' but it still doesn't allow me to reboot the system and reconnect.

### How to Reproduce

Install Debian 12 and rustdesk.
Attempt to remotely restart system.

### Expected Behavior

System restarts and rustdesk reconnects.

### Operating system(s) on local side and remote side

Windows 11 or MacOS -> Debian 12 Stable

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

N/A cannot see login screen after reboot

### Additional Context

The computer is connected to a TV via HDMI. The login screen automatically shows up at 4K resolution but my Debian user modifies this to 1080p. Unsure if this is relevant or useful information.

**Comment by apotenza92** [comment]
I've realised it does connect but it says "failed to obtain screen capture" on the login screen.

**Comment by rustdesk** [comment]
"Login screen" is not supported with Wayland since its security policy limitation. X11 only for login screen.

Actually we have warned you on the RustDesk main window.

**Reply by apotenza92** [reply]
Does this mean there's no support for rebooting on any wayland system?
